### ISSUE TITLE
How to open a  FVP terminal?

### ISSUE THREAD

**Issue by duduscript** [jforissier/secvideo_demo#9]
Hi @jforissier 
I successfully run the step 3 shell script as the readme. But I can not see a FVP terminal in the window that it pops up.  This is the first time I use FVP, and I am confused that I can not see a terminal window.
The Base FVP software model I used is FVP_Base_RevC-2xAEMv8A. Is it the problem of the version of Base FVP software model?
And I had searched for FVP_Base_AEMv8A-AEMv8A, it seems that there is only FVP_Base_RevC-2xAEMv8A provided in arm website.

**Comment by jforissier** [comment]
Hi @duduscript ,

I have used the Armv8-A Foundation Platform package `FM000-KT-00035-r11p4-37rel0.tgz` successfully. You can download it from https://developer.arm.com/products/system-design/fixed-virtual-platforms (registration required).

**Comment by duduscript** [comment]
Hi @jforissier thanks for your reply.
I  download this package but not find a FVP_CMD bin file,
The dirctory lists as follows in Linux64_GCC-4.9:
Foundation_Platform  libFoundation_AEMv8A-Linux64-Release-GCC-4.9.so  libMAXCOREInitSimulationEngine.3.so
libarmctmodel.so     libIrisCore.so                                   libSDL2-2.0.so.0.4.0

Foundation_Platform seems not the FVP_CMD
Is there anything I do not get?

**Comment by jforissier** [comment]
@duduscript I'm sorry, this project is totally obsolete now :(  I have not tried to run it for years (literally). When I answered I mistakenly thought you were trying to use https://github.com/OP-TEE/manifest/blob/master/fvp.xml, which is a different project. The version of OP-TEE that is used here is also very old.

If you are interested in secure data path, I would recommend starting with a current working environment (link above) and write your own application, possibly inspired by code used in this project. There is a feature in optee_os called CFG_SECURE_DATA_PATH. It allows you to share buffers between normal and secure world, the normal world application may manipulate buffers and pass them to Trusted Applications but without being able to read or write into them.

That being said, maybe this toy project can work with the Foundation Platform (the version I linked to does show a CLCD screen on startup).

**Comment by duduscript** [comment]
@jforissier I noticed that this project is obsolete, but there is some part about secure display that I need in my experiment. This project is very close to what I need to do, although I deal with some other problem rather than image display.
I really want to run this project and have tried many version of fvp version. I need a fvp pkg which can run this demo, I will be deeply grateful if you could provide a fvp pkg or a link to download it.

**Comment by jforissier** [comment]
Well, I tried to re-build the code. The toolchain packages are not available any more so I had to update the Makefile a bit and I also changed the default command to be `Foundation_Platform` (the publicly available emulator). Unfortunately, while everything builds OK, the boot hangs shortly after entering BL31.

I've pushed my changes to https://github.com/jforissier/secvideo_demo/tree/foundation in case it is of any help.
